daemon/graphdriver/btrfs/btrfs.go

```diff
@@ -343,7 +335,7 @@ func (d *Driver) updateQuotaStatus() {
 	})
 }
 
-func (d *Driver) subvolEnableQuota() error {
+func (d *Driver) enableQuota() error {
```

Comment:
(Somewhat orthogonally,) If the given path _isn't_ a subvolume, do you think we should have this function error out so we don't accidentally set a quota on `/` thinking we're setting it on something more specific?